src/sas/example_data/media/testdata_help.rst

```diff
@@ -271,6 +300,38 @@ ISIS_98940
 
 .. ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
 
+Magnetic Data
+^^^^^^^^^^^^^
+Polarised SANS data.
+
+Magnetic Test Data
+..................
+**Example data files are located in the \\magnetic_data sub-folder.**
+
+S50
+  - 2D and 1D sector data from 10 nm nanospheres in d8-toluene (c_Fe2O3 = 50 mg/ml,
+    ~1 vol%) at an aligning magnetic field of 1.5 T in horizontal direction.
+  - See `10.1088/1367-2630/14/1/013025 <https://dx.doi.org/10.1088/1367-2630/14/1/013025>`_
+
+.. ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
+
+NR Data
```

Comment:
Should this be renamed to reflectometry data, as there NR and XR data sets?

Reply:
Changed.

Folder name also changed.